src/components/com_tjcertificate/site/views/certificate/tmpl/default_social_sharing.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * @package     TJCertificate
+ * @subpackage  com_tjcertificate
+ *
+ * @author      Techjoomla <extensions@techjoomla.com>
+ * @copyright   Copyright (C) 2009 - 2020 Techjoomla. All rights reserved.
+ * @license     http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL
+ */
+
+// No direct access to this file
+defined('_JEXEC') or die('Restricted access');
+
+use Joomla\CMS\Factory;
+$document = Factory::getDocument();
+?>
+<!-- Social Sharing button start-->
+<?php if ($this->params->get('social_sharing'))
+{
+		// For facebook and linkedin
+		$config = Factory::getConfig();
+		$siteName = $config->get('sitename');
+		$document->addCustomTag('<meta property="og:title" content="' . $this->item->title . '" />');
+		$document->addCustomTag('<meta property="og:image" content="" />');
+		$document->addCustomTag('<meta property="og:url" content="" />');
+		$document->addCustomTag('<meta property="og:description" content="' . $this->item->description . '" />');
+		$document->addCustomTag('<meta property="og:site_name" content="' . $siteName . '" />');
```

Comment:
Also can you please use escape for all the contents before assigning

Reply:
done